### PR TITLE
Quantum Random Number Generator Quickstart

### PR DESCRIPTION
**Pull request by bromeg**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/TOC.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/5da652c3953032df0ee7fe586178427f0474a55b/articles/TOC.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0be6b88b-e275-4e5a-811a-b5eeecc00168_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/5da652c3953032df0ee7fe586178427f0474a55b/articles/quickstarts/qrng.md) | 83 | 54 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9a426d28-16a5-4c5d-9698-1e1afd67a038_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/5da652c3953032df0ee7fe586178427f0474a55b/articles/relnotes/index.md) | 79 | 53 | 124 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fff8e713-973c-4303-8094-6ac4d670028e_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

File: articles/TOC.yml [hunk 1]
```diff
@@ -22,6 +22,8 @@
     href: intro-to-katas.md
 - name: Quickstarts
   items:
+  - name: Build a quantum random number generator
```
Comment (anpaz-msft) [inline articles/TOC.yml]: the title here should match the title on the page.

File: articles/quickstarts/qrng.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            Reset(q);
+            return r;
+            }
+        }
+    }
+    ```
+
+Here we introduce the `Qubit` datatype, native to Q#. We can only allocate a `Qubit` with a `using` statement. When it gets allocated a qubit is always in the `Zero`  state. 
+
+Using the `H` operation, we are able to put our `Qubit` in superposition. To measure a qubit and read its value you use the `M` intrinsic operation.
```
Comment (anpaz-msft) [inline articles/quickstarts/qrng.md]: Ideally, we wouldn't use `H` but have another operation with a more descriptive name.
I'm fine with `M`, as it is quickly recognizable as a shortcut for measure, but `H` is not intuitive (it comes from `Hadamard`, which is not intuitive either).

Also, we probably want to be more specific, what `H` does is puts a qubit in state 0 into a superposition of 0 and 1.

File: articles/quickstarts/qrng.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+title: Create a Quantum Random Number Generator
```
Comment (anpaz-msft) [inline articles/quickstarts/qrng.md]: I would rename this to "Write your First Quantum Program"

File: articles/quickstarts/qrng.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+## Prerequisites
+
+- The Microsoft [Quantum Development Kit](install).
+- [Create a Q# Project](xref:microsoft.quantum.howto.createproject)
+
+
+## Write a Q# operation
+
+As mentioned in our [What is Quantum Computing?](xref:microsoft.quantum.overview.what) article, a qubit is a unit of quantum information that can be in superposition. When measured, a qubit can only be either 0 or 1, however during execution, the state of the qubit represents the probability of reading either a 0 or a 1 during measurement. This probabilistic state is known as superposition. We can use this probability to generate random numbers.
```
Comment (anpaz-msft) [inline articles/quickstarts/qrng.md]: I would move this explanation after the code.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/TOC.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/f71accdd0a85eb99220207595631493745a165d9/articles/TOC.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6881ffe9-f3f0-4570-8e75-f07bc8b02f71_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/f71accdd0a85eb99220207595631493745a165d9/articles/quickstarts/qrng.md) | 77 | 48 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ceab5d69-a69d-4e2a-bc5e-ebaaaf5ef718_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/f71accdd0a85eb99220207595631493745a165d9/articles/relnotes/index.md) | 79 | 53 | 124 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/03d270c8-0589-48fc-9289-d2fd076904f1_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/TOC.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/351bd6725b195bffec518c2ef15b634470568373/articles/TOC.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aacc1f8a-96c8-41e5-81d1-d5249237eda1_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/how-to-guides/create-qsharp-project.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/351bd6725b195bffec518c2ef15b634470568373/articles/how-to-guides/create-qsharp-project.md) | 92 | 71 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bdacb77b-3ea5-4723-8654-eca5b22023db_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/351bd6725b195bffec518c2ef15b634470568373/articles/install-guide/index.md) | 94 | 91 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d999b29d-6db3-454b-8b5e-1b37ec964540_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstart.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/351bd6725b195bffec518c2ef15b634470568373/articles/quickstart.md) | 87 | 66 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/975027f9-48f6-4f8a-936f-dbf8a9e9e92d_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/351bd6725b195bffec518c2ef15b634470568373/articles/quickstarts/qrng.md) | 77 | 48 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f95d2c46-f06b-47ab-814f-e78a632ee066_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/351bd6725b195bffec518c2ef15b634470568373/articles/relnotes/index.md) | 79 | 53 | 124 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/621b9a05-a834-4701-a896-995c7a65d07d_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/TOC.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2f0be7105958296e732035814a1e28e8a70a2374/articles/TOC.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8c4de955-a934-4f1c-b796-33c997889ce0_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/how-to-guides/create-qsharp-project.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2f0be7105958296e732035814a1e28e8a70a2374/articles/how-to-guides/create-qsharp-project.md) | 92 | 72 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b82d737b-e744-46be-a7fb-cf81be7a9210_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2f0be7105958296e732035814a1e28e8a70a2374/articles/install-guide/index.md) | 89 | 77 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aa1dad07-40e6-4873-a7d3-fcd7815588c4_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstart.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2f0be7105958296e732035814a1e28e8a70a2374/articles/quickstart.md) | 86 | 64 | 54 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/51e1921a-7bc7-497a-bedc-201619a88bba_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2f0be7105958296e732035814a1e28e8a70a2374/articles/quickstarts/qrng.md) | 77 | 48 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/96d77acd-2f5f-4782-828c-68301229a57d_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2f0be7105958296e732035814a1e28e8a70a2374/articles/relnotes/index.md) | 79 | 53 | 124 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/99cbd476-1bcb-494e-9848-022f5ca95e43_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Pull request merged**